templates/books-show.html

```diff
@@ -0,0 +1,45 @@
+<div class="row">
+    <div class="col-md-10 col-md-offset-1">
+      <div class="panel panel-default">
+        <div class="panel-body">
+          <div class='row'>
+            <div class="col-md-3 col-xs-12 thumbnail">
+              <img class="book-image" src="{{ bsc.book.image }}" alt="book image">
+            </div>
+            <div class="col-md-9 col-xs-12">
+              <ul class="list-group form-inline">
+                <li class="list-group-item">
+                  <h4 class='inline-header'>Book title:</h4>
+                  <span ng-hide="editing">{{ bsc.book.title }}</span>
+                  <input ng-show="editing" ng-model="bsc.book.title" type="text" class="form-control input-md">
+                </li>
+                <li class="list-group-item">
+                  <h4 class='inline-header'>Author: </h4>
+                  <span ng-hide="editing">{{ bsc.book.author }}</span>
+                  <input ng-show="editing" ng-model="bsc.book.author" type="text" class="form-control input-md">
+                </li>
+                <li class="list-group-item">
+                  <h4 class='inline-header'>Released date:</h4>
+                  <span ng-hide="editing">{{ bsc.book.releaseDate }}</span>
+                  <input ng-show="editing" ng-model="bsc.book.releaseDate" type="text" class="form-control input-md">
+                </li>
+                <li class="list-group-item">
+                  <h4 class='inline-header'>Image:</h4>
+                  <span ng-hide="editing">{{ bsc.book.image }}</span>
+                  <input ng-show="editing" ng-model="bsc.book.image" type="text" class="form-control input-md">
+                </li>
+              </ul>
+            </div>
+          </div>
+          <div class='panel-footer'>
+            <div class='panel-footer'>
+              <button class='btn btn-danger' ng-click="bsc.deleteBook(bsc.book)">Delete Book</button>
+              <button class='btn btn-info' ng-hide="editing" ng-click="editing = true">Edit Book</button>
+              <button class='btn btn-success' ng-show="editing" ng-click="bsc.editBook(bsc.book); editing = false">Save Changes</button>
```

Comment:
should put editing inside of your BooksShowController rather than as a nowhere variable